pkgs/development/python-modules/check-manifest/default.nix

```diff
@@ -14,13 +24,21 @@ buildPythonPackage rec {
     substituteInPlace tests.py --replace "test_build_sdist" "no_test_build_sdist"
   '';
 
-  propagatedBuildInputs = [ build pep517 toml ];
+  propagatedBuildInputs = [
+    build
+    pep517
+    toml
+  ];
 
-  checkInputs = [ mock breezy git pytestCheckHook ];
```

Comment:
The commit message doesn't make sense. It is already using `pytestCheckHook`.

Reply:
The commit message made sense before it was rebased against `staging-next`.